.travis.yml

```diff
@@ -11,27 +11,33 @@ os:
   - linux
   - osx
 
+sudo: required # needed for trusty
+dist: trusty # needed for Raqm
+
 before_install:
   - if [ "$TRAVIS_OS_NAME" = "linux" ];
     then
+      sudo apt-add-repository -y ppa:as-bahanta/raqm;
```

Comment:
So what are the alternatives? I fear that the code will easily bitrot if it is not actively built and tested.